java/runtime/src/main/java/io/ray/runtime/placementgroup/PlacementGroupImpl.java

```diff
@@ -30,33 +31,34 @@ private PlacementGroupImpl(
     this.state = state;
   }
 
+  @Override
   public PlacementGroupId getId() {
     return id;
   }
 
+  @Override
   public String getName() {
     return name;
   }
 
+  @Override
   public List<Map<String, Double>> getBundles() {
     return bundles;
   }
 
+  @Override
   public PlacementStrategy getStrategy() {
     return strategy;
   }
 
+  @Override
   public PlacementGroupState getState() {
     return state;
   }
 
-  /**
-   * Wait for the placement group to be ready within the specified time.
-   *
-   * @param timeoutSeconds Timeout in seconds.
-   * @return True if the placement group is created. False otherwise.
-   */
-  public boolean wait(int timeoutSeconds) {
+  @Override
+  public boolean wait(long duration, TimeUnit unit) {
+    int timeoutSeconds = Math.toIntExact(TimeUnit.SECONDS.convert(duration, unit));
```

Comment:
What if a user wants to wait for only half a second?

Reply:
Has discussed with @kfstorm , will remove the TimeUnit setup.